airbyte-integrations/connectors/destination-snowflake/src/test/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationTest.java

```diff
@@ -92,35 +75,6 @@ public void useInsertStrategyTest() {
     assertFalse(SnowflakeDestinationResolver.isS3Copy(stubConfig));
   }
 
-  @Test
-  public void testCleanupStageOnFailure() throws Exception {
```

Comment:
This test has been removed because we no longer attempt to `copyIntoTmpTableFromStage` within the `onCloseFunction` when no failure occurs while flushing buffer, this is done within the `flushBufferFunction` where if a failure occurs, the failure is tracked within the `hasFailed` variable and regardless of failure or success the logic will execute `dropStageIfExists` if the user specifies they want to `purgeStagingData`

`dropStageIfExists` will clear out the entire bucket which will supersede cleaning up individual staging files